setup.py

```diff
@@ -135,7 +139,25 @@ def get_extensions():
             include_dirs=tests_include_dirs,
             define_macros=define_macros,
             extra_compile_args=extra_compile_args,
-        )
+        ),
+        CppExtension(
+            'torchvision.video_reader',
+            video_reader_src,
+            include_dirs=[
+                video_reader_src_dir,
+                '/home/zyan3/local/anaconda3/envs/pytorch_py3/include',
```

Comment:
@fmassa , I will remove this line.

For header files from ffmpeg, we need to ensure they are installed at default header file search path.